### ISSUE TITLE
UI suggestion: long press to reorder activities

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [Medvedev91/timeto.me#64]
Hi,

I think that the current way to reorder activities is quite cumbersome: it makes it very awkward up move anactivity to the top. The android pattern where long pressing on one element allows dragging it seems appropriate here.

**Comment by gnurlbash** [comment]
I agree with this. In addition, the buttons in the Edit activity screen are too close together and it's way too easy to press the wrong one if you're not careful.

On a somewhat related topic, I wish there was a way to reorder repeating tasks. As far as I can see, they are just piled on top of each other in order of creation and there's no way to move them up or down like in the other screen.

**Comment by Medvedev91** [comment]
> I think that the current way to reorder activities is quite cumbersome

Someday I will implement this.

> As far as I can see, they are just piled on top of each other in order of creation

They are sorted by time of day or day they are due. I don't think that needs to change.
